src/coredump/coredump.c

```diff
@@ -1712,12 +1712,6 @@ static int gather_pid_mount_tree_fd(const Context *context) {
 
         pair[1] = safe_close(pair[1]);
 
-        r = wait_for_terminate_and_check("(sd-mount-tree-ns)", child, 0);
-        if (r < 0)
-                return log_error_errno(r, "Failed to wait for child: %m");
-        if (r != EXIT_SUCCESS)
-                return log_error_errno(SYNTHETIC_ERRNO(ECHILD), "Child died abnormally.");
```

Comment:
the removal of the local `child` variable should really be squashed into this commit (ended up in the next)

Reply:
not addressed.